elfpy/__init__.py

```diff
@@ -30,7 +30,7 @@
 PRECISION_THRESHOLD: FixedPoint = FixedPoint(1 * 10**10)  # 1e-8 * 1e18 = 1e10
 
 # Logging defaults
-DEFAULT_LOG_LEVEL = logging.INFO
+DEFAULT_LOG_LEVEL = "INFO"
```

Comment:
why not use logging.INFO here?

Reply:
cus that's a number and puts `"log_level": 20,` in `my_config.json` leaving me puzzled (I had to look it up)